src/utilities/indexer/updateAttestationsCreated.ts

```diff
@@ -0,0 +1,64 @@
+import { type ICType } from '@kiltprotocol/sdk-js';
+
+import { Op } from 'sequelize';
+
+import { CType as CTypeModel } from '../../models/ctype';
+
+import { logger } from '../logger';
+
+import { matchesGenerator, QUERY_SIZE } from './queryFromIndexer';
+
+// When modifying queries, first try them out on https://indexer.kilt.io/ or https://dev-indexer.kilt.io/
+
+function buildQueriesForAttestationsCreated() {
+  return (offset: number) => `
+      query {
+        attestationsCreated: cTypes(orderBy: ID_ASC, first: ${QUERY_SIZE}, offset: ${offset}) {
+          totalCount
+          nodes {
+            cTypeId: id
+            attestationsCreated
+            registrationBlockId
+          }
+        }
+      }
+    `;
+}
```

Comment:
```suggestion
function buildQueriesForAttestationsCreated(offset: number) {
  return `
      query {
        attestationsCreated: cTypes(orderBy: ID_ASC, first: ${QUERY_SIZE}, offset: ${offset}) {
          totalCount
          nodes {
            cTypeId: id
            attestationsCreated
            registrationBlockId
          }
        }
      }
    `;
}
```
Could this be simpler?

Reply:
Fair enough!